src/gui/guiFormSpecMenu.cpp

```diff
@@ -507,7 +534,8 @@ void GUIFormSpecMenu::parseCheckbox(parserData* data, const std::string &element
 			Environment->setFocus(e);
 		}
 
-		m_checkboxes.emplace_back(spec,e);
+		e->grab();
```

Comment:
Because of the complaints of stu, all pointers in `m_checkboxes`, `m_inventorylists` and co. are dropped when these containers are cleared. The `IGUICheckBox *` was not made with `new` but with an `add…` function. All pointers in `m_checkboxes` and co. are not only grabbed by the environment but also by the `GUIFormSpecMenu` code.